src/routes/transactions/tests/routes.rs

```diff
@@ -0,0 +1,429 @@
+extern crate dotenv;
+
+use crate::common::models::page::Page;
+use crate::config::{
+    chain_info_request_timeout, contract_info_request_timeout, safe_info_request_timeout,
+    token_info_request_timeout, transaction_request_timeout,
+};
+use crate::providers::info::TokenInfo;
+use crate::routes::transactions::models::details::TransactionDetails;
+use crate::routes::transactions::tests::{MULTISIG_TX_DETAILS, POST_CONFIRMATION_RESULT};
+use crate::tests::main::setup_rocket;
+use crate::utils::errors::{ApiError, ErrorDetails};
+use crate::utils::http_client::{MockHttpClient, Request, Response};
+use core::time::Duration;
+use mockall::predicate::eq;
+use rocket::http::{ContentType, Header, Status};
+use rocket::local::asynchronous::Client;
+use serde_json::json;
+
+#[rocket::async_test]
+async fn post_confirmation_success() {
+    let mock_http_client = {
+        let mut mock_http_client = MockHttpClient::new();
+
+        let mut chain_request = Request::new(config_uri!("/v1/chains/{}/", 4));
+        chain_request.timeout(Duration::from_millis(chain_info_request_timeout()));
```

Comment:
What happens if we don't have this timeout with the mock? This should be negligible right?

Reply:
It will fail to match the `Request` object when trying to find the correct mock for the `get` method call. It matches the whole instance of `Request`.